### ISSUE TITLE
Tracking Issue - Accessibility Review: Bounties Landing Page

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1040]
# Accessibility Review: Bounties Landing Page - WIP

**URL:** [https://community.sphinx.chat](https://community.sphinx.chat)  
**Objective:** Identify and track accessibility issues on the Bounties Landing Page.  

Applicable Guidelines: [WCAG 2.2](https://www.w3.org/TR/WCAG22/) & [WAI-ARIA 1.2](https://www.w3.org/TR/wai-aria-1.2/) as baseline; [APCA](https://www.myndex.com/APCA/) & [ISO 9241-210:2019](https://www.iso.org/standard/77520.html) supplemental

## 📌 Identified Issues:
- [ ] **Lack of visible keyboard focus on controls** ([#issue_number](#)) 
- [ ] **Hamburger nav "menu" button missing required properties and states (e.g.`aria-expanded`)** ([#issue_number](#)) 
- [ ] **Missing Landmark Regions (Banner, Main, Nav in footer, etc.) [Landmarks - Scott O'Hara](https://www.scottohara.me/blog/2018/03/03/landmarks.html)** ([#issue_number](#))  
- [ ] **Low contrast on CTA buttons [(Fails both WCAG2 & APCA)](https://www.myndex.com/BPCA/?BG=608aff&TXT=ffffff&DEV=G4g&BUF=A22)** ([#issue_number](#))
- [ ] **"Sign in" button name, role, value and focus order issues** ([#issue_number](#))   
- [ ] **Incorrect alt text for card images** ([#issue_number](#))  
- [ ] **List markup missing from nav content (header, footer, expanded mobile)** ([#issue_number](#))  
- [ ] **Lack of audio description or transcript for video** ([#issue_number](#))  
- [ ] **Auto-playing audio/video (Best Practice)** ([#issue_number](#))  

## 🔄 Next Steps:
1. Create individual issues for each item above and link them here.
2. Implement fixes based on acceptance criteria.
3. Review and validate changes.

**Comment by jason-me** [comment]
@humansinstitute - I am clear to start most of these with the exception of the contrast issue which will likely require a design decision and the audio description, which may involve professionally recording audio. 

I have already written up code changes for the visible focus issues and sign-in button  from a PR that was super-ceded that I can re-purpose.

- [ ] **Lack of visible keyboard focus on controls** ([Issue #1042](https://github.com/stakwork/sphinx-tribes-frontend/issues/1042)) 
- [ ] **"Sign in" button name, role, value and focus order issues** ([Issue #1047](https://github.com/stakwork/sphinx-tribes-frontend/issues/1047))

**Comment by jason-me** [comment]
Related PR to close sub issues #1044, #1045, #1048. and #1049: https://github.com/stakwork/sphinx-tribes-frontend/pull/1140
